pkg/kubelet/kuberuntime/kuberuntime_container_linux_test.go

```diff
@@ -299,8 +299,16 @@ func TestCalculateLinuxResources(t *testing.T) {
 		},
 	}
 	for _, test := range tests {
-		linuxContainerResources := m.calculateLinuxResources(test.cpuReq, test.cpuLim, test.memLim)
-		assert.Equal(t, test.expected, linuxContainerResources)
+		resources := m.calculateLinuxResources(test.cpuReq, test.cpuLim, test.memLim)
+
+		if resources.Unified["memory.oom.group"] != "" {
+			if test.expected.Unified == nil {
+				test.expected.Unified = map[string]string{}
+			}
+			test.expected.Unified["memory.oom.group"] = resources.Unified["memory.oom.group"]
```

Comment:
actually

/lgtm cancel

instead of setting the expected to the value of the actual, can you set expected to `1` to verify the actual is correct?

`test.expected.Unified["memory.oom.group"] = "1"`

Reply:
Done.